modules/cnf-assigning-a-sriov-network-to-a-vrf.adoc

```diff
@@ -0,0 +1,77 @@
+[id="cnf-assigning-a-sriov-network-to-a-vrf"]
+= Assigning an SR-IOV network to a VRF
+
+As a cluster administrator, you can assign an SR-IOV network interface to your VRF domain by using the VRF CNI plug-in.
+
+To do this, you must add the VRF configuration to the optional metaPlugins parameter of the `SriovNetwork` resource.
+
+== Creating an additional SR-IOV network attachment with the VRF CNI plug-in
+
+The SR-IOV Network Operator manages additional network definitions. When you specify an additional SR-IOV network to create, the SR-IOV Network Operator creates the NetworkAttachmentDefinition custom resource (CR) automatically.
+
+NOTE: Do not edit the `NetworkAttachmentDefinition` CRs that the SR-IOV Network Operator manages. Doing so might disrupt network traffic on your additional network.
+
+=== Prerequisites
+
+* Install the {product-title} CLI (oc).
+* Log in to the {product-title} cluster as a user with cluster-admin privileges.
+
+=== Procedure
+
+. Edit the `SriovNetwork` CR by running the following command:
+
+[source,terminal]
+----
+$ oc edit sriovnetwork.openshift.io cluster
+----
+
+. Modify the CR that you are creating by adding the `metaPlugins` configuration for the additional network you are creating, as in the following example CR.
```

Comment:
"extend the cr by ..." (because we are creating it)

Reply:
fixed in latest commit